homeassistant/components/bring/todo.py

```diff
@@ -103,27 +114,42 @@ async def async_update_todo_item(self, item: TodoItem) -> None:
         """Update an item to the To-do list.
 
         Bring has an internal 'recent' list which we want to use instead of a todo list
-        status, therefore completed todo list items will directly be deleted
+        status, therefore completed todo list items are matched to the recent list and pending items to the purchase list
```

Comment:
Please limit docstring lines to max 72 characters per line in accordance with PEP8.

Reply:
Please end sentences with a period.

Reply:
@MartinHjelmare Follow up PR: https://github.com/home-assistant/core/pull/111276